.rulesync/rules/project.md

```
@@ -0,0 +1,172 @@
+
+## Project Conventions
+
+### Code Style
+- ソースコードにはロジックの内容がわかるように日本語のコメントをいれること
+
+- 値オブジェクト（Value Object）としての Newtype パターン
+Rustのタプル構造体を使って、型安全性を担保します。プリミティブ型（i32やString）の直接利用を避けます。
+
+Rust
+// domain/model/unit.rs
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
```

markdownlint-cli2 (0.21.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)